.github/scripts/trymerge.py

```diff
@@ -1397,6 +1402,11 @@ def merge(pr_num: int, repo: GitRepo,
     # here to stop the merge process right away
     find_matching_merge_rule(pr, repo, skip_mandatory_checks=True)
 
+    if not has_required_labels(pr):
+        raise RuntimeError(LABEL_ERR_MSG.lstrip(" #"))
+    else:
+        delete_all_label_err_comments(pr)
```

Comment:
This part can be left out from here to simplify the logic.  The deletion was there so that if someone pushed 5 commits to their PR, they wouldn't end up with 5 instances of the "please add a label" comment